packages/ecc-utils-design/package.json

```diff
@@ -17,23 +16,23 @@
     "./dist/utilities/*": "./dist/utilities/*"
   },
   "type": "module",
-  "version": "0.0.1",
-  "author": "",
-  "license": "ISC",
-  "customElements": "./dist/custom-elements.json",
-  "repository": "",
-  "componentsPrefix": "ecc-utils-design-",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elixir-cloud-aai/cloud-components.git"
+  },
+  "customElements": "dist/custom-elements.json",
+  "files": ["dist", "README.md", "package.json", "LICENSE"],
```

Comment:
maybe a nit pick but since we already have such huge changes, why not format these small things as well. Maybe adding a linter would have been a good idea, but as you guy think that it can a separate issue then ig so.

But regardless can we still format it with our IDE's and will add the formatter later.
PS: prettier is a dep already in root dir.